### ISSUE TITLE
Don't always force small arg1 onto the first line in multiline function call

### ISSUE THREAD

**Issue by glenjamin** [avh4/elm-format#178]
Having the first argument on the same line as the function implies that the first argument is somehow more important - which in this example is not the case:

```
initialWindowSize : Cmd Msg
initialWindowSize =
    Task.perform (\_ -> Debug.crash "no window?")
        WindowSize
        Window.size
```

Is there any better way to write this?

**Comment by rtfeldman** [comment]
Assuming you don't want it all on one line, I think it's pretty clear that there is a better way to write this:

```elm
initialWindowSize : Cmd Msg
initialWindowSize =
    Task.perform
        (\_ -> Debug.crash "no window?")
        WindowSize
        Window.size
```

However, always formatting multiline function application like this led to https://github.com/avh4/elm-format/issues/115, which I think is a way worse outcome than the original example.

So I think the real question is not "is there any better way to write this," because there seems to be a pretty obvious answer to that, nor is it "should we always break before the first argument" (see #115 for why not), but rather "should end users have the power to choose between breaking before or after the first argument in function application?"

If they should, this would be the only case in elm-format (that I'm aware of) where users have that choice. I'm pretty sure everywhere else you just choose Single Line or Multiline and that's it.

**Comment by rtfeldman** [comment]
Personally my thoughts are as follows:

👍 Status quo as of 0.3.0
👍 Introduce a choice for this
👎 Go back to 0.2.0 status quo

Either of the first two are fine by me, and I don't have a strong preference between them.

**Comment by avh4** [comment]
I'm not 100% sure this is what we want, but 0.4.0-alpha will allow three formattings: all on one line; function and first argument on one line and other arguments on their own lines; and all arguments on their own lines.  Let's see how this seems after using it for a while.

Fixed in fba3025..a166813.
